{
  "success": false,
  "run_id": 0,
  "eval": null,
  "duration_in_s": -1.0,
  "agent_answer": "Task failed due to Page.wait_for_selector: Timeout 60000ms exceeded.\nCall log:\n  - waiting for locator(\"body\") to be visible\n  -     123 \u00d7 locator resolved to hidden <body class=\"mobile\">\u2026</body>\n Traceback (most recent call last):\n  File \"/Users/apinto/dev/nottelabs/notte/src/notte_eval/run.py\", line 192, in run_agent\n    run = await agent_bench.run_agent(task)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/src/notte_eval/agent_handlers/convergence.py\", line 110, in run_agent\n    result = await agent.run(prompt)\n             ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/src/notte_eval/patcher.py\", line 123, in async_wrapper\n    result = await func(*args, **kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/proxy_lite/runner.py\", line 171, in run\n    async for run in self.run_generator(task):\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/proxy_lite/runner.py\", line 161, in run_generator\n    observation: Observation = await environment.execute_action(action)\n                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/proxy_lite/environments/webbrowser.py\", line 155, in execute_action\n    original_img, annotated_img = await self.browser.screenshot(\n                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/proxy_lite/browser/browser.py\", line 280, in screenshot\n    await self.update_poi()\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/tenacity/asyncio/__init__.py\", line 189, in async_wrapped\n    return await copy(fn, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/tenacity/asyncio/__init__.py\", line 111, in __call__\n    do = await self.iter(retry_state=retry_state)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/tenacity/asyncio/__init__.py\", line 153, in iter\n    result = await action(retry_state)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/tenacity/_utils.py\", line 99, in inner\n    return call(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/tenacity/__init__.py\", line 418, in exc_check\n    raise retry_exc.reraise()\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/tenacity/__init__.py\", line 185, in reraise\n    raise self.last_attempt.result()\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/homebrew/Cellar/python@3.11/3.11.10/Frameworks/Python.framework/Versions/3.11/lib/python3.11/concurrent/futures/_base.py\", line 449, in result\n    return self.__get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/homebrew/Cellar/python@3.11/3.11.10/Frameworks/Python.framework/Versions/3.11/lib/python3.11/concurrent/futures/_base.py\", line 401, in __get_result\n    raise self._exception\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/tenacity/asyncio/__init__.py\", line 114, in __call__\n    result = await fn(*args, **kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/proxy_lite/browser/browser.py\", line 204, in update_poi\n    await self.current_page.wait_for_selector(\"body\", timeout=60000, state=\"visible\")\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/playwright/async_api/_generated.py\", line 8176, in wait_for_selector\n    await self._impl_obj.wait_for_selector(\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/playwright/_impl/_page.py\", line 425, in wait_for_selector\n    return await self._main_frame.wait_for_selector(**locals_to_params(locals()))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/playwright/_impl/_frame.py\", line 323, in wait_for_selector\n    await self._channel.send(\"waitForSelector\", locals_to_params(locals()))\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/playwright/_impl/_connection.py\", line 61, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apinto/dev/nottelabs/notte/.venv/lib/python3.11/site-packages/playwright/_impl/_connection.py\", line 528, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nplaywright._impl._errors.TimeoutError: Page.wait_for_selector: Timeout 60000ms exceeded.\nCall log:\n  - waiting for locator(\"body\") to be visible\n  -     123 \u00d7 locator resolved to hidden <body class=\"mobile\">\u2026</body>\n\n",
  "task": {
    "question": "Search for the latest news title about the NBA team the Los Angeles Lakers.",
    "id": "webvoyager--Google Search--2",
    "answer": "News Title (real-time)",
    "url": "https://www.google.com/",
    "website_name": "Google Search"
  },
  "steps": [],
  "logs": {},
  "task_description": "Search for the latest news title about the NBA team the Los Angeles Lakers.",
  "task_id": "webvoyager--Google Search--2",
  "task_website": "Google Search",
  "reference_answer": "News Title (real-time)",
  "total_input_tokens": 0,
  "total_output_tokens": 0,
  "last_message": ""
}